contracts/base/ModuleManager.sol

```diff
@@ -153,10 +153,10 @@ abstract contract ModuleManager is SelfAuthorized, Executor, IModuleManager {
         bytes memory data,
         Enum.Operation operation
     ) public override returns (bool success) {
+        onBeforeExecTransactionFromModule(to, value, data, operation);
```

Comment:
This should move into the `preModuleExecution` if possible. (I would double-check gas characteristics, but I imagine them to be equivalent.

Reply:
After moving code:

Saves `12` additional bytes compared to direct call from `execTransactionFromModule*`
```
Safe 21832 bytes (limit is 24576)
SafeL2 22612 bytes (limit is 24576)
```

Gas usage doesn't change

```
·------------------------------------------------------------|----------------------------|-------------|------------------------------·
|                    Solc version: 0.7.6                     ·  Optimizer enabled: false  ·  Runs: 200  ·  Block limit: 100000000 gas  │
·····························································|····························|·············|·······························
|  Methods                                                                                                                             │
·····················|·······································|·············|··············|·············|···············|···············
|  Contract          ·  Method                               ·  Min        ·  Max         ·  Avg        ·  # calls      ·  usd (avg)   │
·····················|·······································|·············|··············|·············|···············|···············
|  Safe              ·  execTransaction                      ·      51999  ·     3638489  ·     133049  ·          248  ·           -  │
·····················|·······································|·············|··············|·············|···············|···············
|  Safe              ·  execTransactionFromModule            ·      51625  ·      184230  ·     127862  ·           11  ·           -  │
·····················|·······································|·············|··············|·············|···············|···············
|  Safe              ·  execTransactionFromModuleReturnData  ·      52268  ·      183974  ·     107914  ·            5  ·           -  │
·····················|·······································|·············|··············|·············|···············|···············
```

So, moved the code as suggested here: https://github.com/safe-global/safe-smart-account/pull/776/commits/f827f9122e1eb12a87bcb8831b3a3ed8af3c0b42